lib/Sema/TypeCheckStmt.cpp

```diff
@@ -1333,6 +1336,19 @@ void TypeChecker::checkIgnoredExpr(Expr *E) {
 
 Stmt *StmtChecker::visitBraceStmt(BraceStmt *BS) {
   const SourceManager &SM = TC.Context.SourceMgr;
+
+  // Diagnose defer statement being last one in block ( Only if
+  // BraceStmt does not start a TopLevelDecl )
```

Comment:
Formatting nit: please end comments with a period here and elsewhere; also, no spaces or capitalization inside parentheses.

Reply:
Fixed in next commit.